RHTE-LAB.adoc

```diff
@@ -10,30 +10,480 @@ This repo should act as a resource for cluster administrators to begin the proce
 
 In OpenShift 4, every cluster starts out the same way -- a bootstrapped control plane with master, and worker nodes.
 However, in order for this cluster to provide value to an organization, it needs further configuration.
-Using this repository as a starting point we are going to establish an automated strategy for managing configuration of an operationalized cluster, including things like:
-
-* Enabling Cluster Autoscaling
-* Configuring authentication and authorization
-* Configuring namespace creation
-* Setting up quotas & limits for applications
-* Applying automated certificate management for applications
-* Deploying initial workloads
-* Deploying custom dashboards and setting up alerts
 
```

Comment:
In this lab, we are going to develop a strategy for making these configurations and maintaining them over time. We are going to employ the concepts of link:https://openpracticelibrary.com/practice/everything-as-code/[Everything as Code] and link:https://thenewstack.io/what-is-gitops-and-why-it-might-be-the-next-big-thing-for-devops/[GitOps] to create _self-managing_ OpenShift clusters, where the day 2 operations are done through link:https://www.weave.works/blog/gitops-operations-by-pull-request[Pull Requests].